qa/tasks/cephfs/test_nfs.py

```diff
@@ -487,6 +488,88 @@ def update_export(self, cluster_id, path, pseudo, fs_name):
                                  }
                              }))
 
+    def enable_libcephfs_logging(self, cluster_name: str):
+        fsid = self._cmd("fsid").strip()
+        log.debug(f"get fsid: {fsid}")
+        # add volume mount for ceph client logging from /var/log/ceph/$fsid:/var/log/ceph:z
+        ganesha_spec = self._cmd("orch", "ls", "--service-name",
+                                 f"nfs.{cluster_name}", "--export").strip()
+        parsed_ganesha_spec = yaml.safe_load(ganesha_spec)
+        original_ganesha_spec = yaml.dump(parsed_ganesha_spec)
+        parsed_ganesha_spec["extra_container_args"] = ["-v",
+                                                       f"/var/log/ceph/{fsid}:/var/log/ceph:z"]
+        debug_enabled_ganesha_spec = yaml.dump(parsed_ganesha_spec).replace("- -v", '- "-v"').replace(
+            f"- /var/log/ceph/{fsid}:/var/log/ceph:z", f'- "/var/log/ceph/{fsid}:/var/log/ceph:z"')
+        log.debug(f"debug_enabled_ganesha_spec: {debug_enabled_ganesha_spec}")
+        self.ctx.cluster.run(
+            args=['ceph', 'orch', 'apply', '-i', '-'],
+            stdin=debug_enabled_ganesha_spec)
```

Comment:
This looks like an evil hack but I guess you tested it works. @adk3798 any opinions?

Reply:
I think at some point we should add some actual changes to the nfs and cephadm modules to support this properly. For the time being, since we aren't making the spec ourselves and are instead using the nfs module to set up the nfs service (I believe, I didn't actually check) this modification of the spec after the fact is what you'd have to do.

Reply:
> This looks like an evil hack but I guess you tested it works. @adk3798 any opinions?

Yup, it works and not just this part but this entire PR is a hack haha but I can't think of a better stop gap than this (at least we have logs in teuthology). Eventually a long term fix should come under https://tracker.ceph.com/issues/69895.